packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -6,11 +6,162 @@
  * @prop --calcite-alert-width: Specifies the width of the component.
  */
 
+:host {
+  --calcite-alert-edge-distance: theme("spacing.8");
+  @apply block;
+}
+
+.container {
+  @apply bg-foreground-1
+  box-border
+  fixed
+  flex
+  flex-wrap
+  items-center
+  justify-center
+  mx-auto
+  my-0
+  opacity-0
+  pointer-events-none
+  shadow-2
+  text-start
+  w-full
+  z-toast;
+
+  container: responsive-container / inline-size;
+  border-radius: var(--calcite-border-radius);
+  border-block-start: 0 solid transparent;
+  border-inline: 1px solid var(--calcite-ui-border-3);
+  border-block-end: 1px solid var(--calcite-ui-border-3);
+  inline-size: var(--calcite-alert-width);
+  max-inline-size: calc(100% - (var(--calcite-alert-edge-distance) * 2 + 2px));
```

Comment:
What is the `2px` here from? can that come from a token?

Reply:
Agreed on using internal CSS props or Sass vars for these. @macandcheese could you help us out with this?

Reply:
Hm, this is an older one, maybe to compensate for the 1px border on either side? May not be necessary anymore.

Reply:
Doesn't seem to be necessary. Removed!